Sources/Track.swift

```diff
@@ -139,32 +141,32 @@ class Track {
         update(&superProperties)
     }
 
-    func time(event: String?, timedEvents: InternalProperties, startTime: Double) -> InternalProperties {
+    func time(eventID: TimedEventID, timedEvents: TimedEvents, startTime: TimeInterval) -> TimedEvents {
```

Comment:
I renamed the property name for clarity, changed the types for consistency/correctness and removed the optionality of the eventID because it didn't actually need to be optional.